tests/compiler/std/arraybuffer.optimized.wat

```diff
@@ -1267,6 +1267,11 @@
     call $~lib/rt/tlsf/searchBlock
     local.tee $2
     i32.eqz
+    if
+     unreachable
+    end
+    local.get $2
+    i32.eqz
```

Comment:
That looks redundant:
```
if (!$2) unreachable();
if (!$2) abort();
```

Reply:
Right, there's a defensive assert (the second one) in there because this wasn't implemented yet. Taking a look